tests/unit/test_charm.py

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env python3
+"""
```

Comment:
Can you add the docstring after the license? Also, why a multiline comment

Reply:
Hey sorry, I need to update the PR description - I split this PR into multiple ones (one for Jira integration, one for Trivy, one for addressing linting issues) and what you mention is addressed in the feat/linters PR. When it lands on main, I'll sync and this PR will have only chaos test related changes like it was supposed to be in the beginning :)